wlauto/instrumentation/fps/__init__.py

```diff
@@ -221,23 +261,40 @@ def __init__(self, outfile, device, activities, keep_raw, logger, dumpsys_period
         self.frames = []
         self.last_ready_time = 0
         self.refresh_period = VSYNC_INTERVAL
-        self.drop_threshold = self.refresh_period * 1000
+        self.drop_threshold = self.refresh_period * 10
```

Comment:
Code sets drop_threshold to *10 when reading in the refresh_period from _process_surfaceflinger_line anyway. Changed to reflect the same as 'normal behaviour' so I can avoid doing this in _process_gfxinfo_line explicitly.
Where does * 1000 come from and why is it never used?